### ISSUE TITLE
USA - SAW total American victory broken

### ISSUE THREAD

**Issue by GoldenShowa** [TNODevs/TNO-Bug-Tracker#1994]
### Operating System

Windows 10

### HOI4 version

1.12.14

### TNO version

1.4.1d

### Do you have other mods in use?

none

### Were you using Steam?

Yes

### Which major expansions do you NOT have?

I have all

### Describe the bug

The South African war total victory for the US event doesn't fire and the war continues even though all African RKs have been capitulated

### Steps to reproduce / Cause of the issue

1. Play as the USA
2. Achieve total victory in the SAW
3. Event that would usually fire to signal the end of the war and the establishment of the mandates does not appear

### Expected behaviour

SAW victory event to fire and the event to establish the mandates to appear

### Screenshots

![tnobug2](https://github.com/TNODevs/TNO-Bug-Tracker/assets/94769222/dc87040c-04c1-4b87-ab45-36e8abf2af23)

**Comment by Victorrah** [comment]
i have the same problem, if you type event SAFWAR.11110 into console it removes the SAW from the global conflicts thing and gives you an ending event even though its outdated from like 3 years ago, it also dosnt release mandates so hopefully someone finds a fix

**Comment by TokeVFA** [comment]
Here to confirm same behavior, Won war, nothing happened.
If i use console events. I can make the war end, and the rest of the event trigger, however the madates do not spawn, the west and central mandate cores do appear, but the contries do not.

if i use the events to swap to the mandates the games end in game over as you have no lands.

**Comment by runelead00** [comment]
#1509 - Saves prior to the peace deal and the relative logs are needed to fix peace deals

**Comment by Golden-Pigeon** [comment]
I have the same problem. Here are related saves and logs.

[SAW.zip](https://github.com/TNODevs/TNO-Bug-Tracker/files/11937000/SAW.zip)

**Comment by kendallizer** [comment]
Hello! Recently ran into this issue again.

I fixed it by using debug before the peace deal happens, and just pressing Total OFN victory. It gives the mandates and their related mechanics seamlessly. 

However, if you this issue is still open, I would like to contribute the save file. The only other mod I have installed is Radio TNO. This is my first time uploading a file here, so I apologize if I've done it incorrectly.

Some notes - the run is cheated pretty hard, but only through commands such as pp, cp, delall, and debug. Before this, no debug options were used. The peace-deal should fire within a few days of this save, it's not exactly right before, but it sometimes can be depending on SA and its forces capturing a VP. 

[SAF TOTAL VICTORY.zip](https://github.com/TNODevs/TNO-Bug-Tracker/files/12600016/SAF.TOTAL.VICTORY.zip)
